src/audio/copier/copier_gain.h

```diff
@@ -95,6 +95,15 @@ struct copier_gain_params {
 	uint16_t chanels_count; /**< Number of channels */
 };
 
+/** Gain Coefficients IO Control
+ *
+ * This parameter is sent by the driver to add/modify a static gain.
+ * Coefficients are encoded in Q10 format.
+ */
+struct gain_dma_control_data {
+	uint16_t gain_coeffs[MAX_GAIN_COEFFS_CNT];
+} __packed __aligned(4);
```

Comment:
what is the relationship between a gain control and a DMA?

This is not self-explanatory I am afraid.